core/src/lib.rs

```diff
@@ -244,6 +244,8 @@ impl Attribute {
         attribute_map.insert(values::DB_CARDINALITY.clone(), if self.multival { values::DB_CARDINALITY_MANY.clone() } else { values::DB_CARDINALITY_ONE.clone() });
 
         if self.unique == Some(attribute::Unique::Value) {
+            attribute_map.insert(values::DB_UNIQUE.clone(), values::DB_UNIQUE_VALUE.clone());
+        } else if self.unique == Some(attribute::Unique::Identity) {
```

Comment:
```rust
match self.unique {
    Some(attribute::Unique::Value) => {
        attribute_map.insert…
    },
    Some(attribute::Unique::Identity) => …,
    None => {},
}
```

If we later add an additional type of uniqueness, the `match` form will tell you about an unhandled case — which it would also have here if you missed out `Identity`.